index.js

```diff
@@ -1,10 +1,12 @@
 var textParsers = require('./lib/textParsers');
 var binaryParsers = require('./lib/binaryParsers');
 var arrayParser = require('./lib/arrayParser');
+var builtinTypes = require('./lib/builtins');
 
 exports.getTypeParser = getTypeParser;
 exports.setTypeParser = setTypeParser;
 exports.arrayParser = arrayParser;
+exports.builtinTypes = builtinTypes;
```

Comment:
Can you make this `exports.builtins` so that you refer to them as `types.builtins.TIMESTAMPTZ`? Otherwise this is great, thank you!

Reply:
Done.